src/sections/contact.html

```diff
@@ -1,3 +1,41 @@
-<div>
-   <p>Contact</p>
-</div>
\ No newline at end of file
+<section class="flex flex-col gap-36 pt-64">
+  <div
+    class="mx-auto flex-col px-3 text-customGreen lg:flex lg:w-8/12 lg:flex-row lg:gap-36"
+  >
+    <div class="flex w-[65%] flex-col gap-y-10 text-3xl">
```

Comment:
Its a good coding practice to avoid using hard coded values. it would be better if you configure the width dimension in tailwind config.